src/drc/tests/test_userstory_609_vertrouwelijkheidaanduiding.py

```diff
@@ -0,0 +1,95 @@
+"""
+Test value of vertrouwelijkheidaanduiding while creating EnkelvoudigInformatieObject
+
+See:
+https://github.com/VNG-Realisatie/gemma-zaken/issues/609
+"""
+from base64 import b64encode
+
+from django.test import override_settings, tag
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+from zds_client.tests.mocks import mock_client
+from zds_schema.constants import VertrouwelijkheidsAanduiding
+from zds_schema.tests import JWTScopesMixin, reverse
+
+from drc.api.scopes import SCOPE_DOCUMENTEN_ALLES_VERWIJDEREN
+
+
+@override_settings(
+    LINK_FETCHER='zds_schema.mocks.link_fetcher_200',
+    ZDS_CLIENT_CLASS='zds_schema.mocks.MockClient'
+)
+class US6609TestCase(JWTScopesMixin, APITestCase):
+
+    scopes = [SCOPE_DOCUMENTEN_ALLES_VERWIJDEREN]
```

Comment:
scope to delete everything seems like too many permissions for the actual behaviour we're testing :smile: